docs/guides/miscellaneous/migration_guide.md

```diff
@@ -263,6 +315,80 @@ Suite Name                                   Status     Expectations met
+  
+  If you choose to do so, the `citation` metadata can be migrated using the same pattern for migrating `batch_kwargs` to `batch_request` described below. 
+
+:::
+
+#### V2-Style Checkpoint
+
+```yaml file=../../../tests/test_fixtures/configuration_for_testing_v2_v3_migration/spark/v2/great_expectations/checkpoints/test_v2_checkpoint.yml#L1-L15
+```
+
+The Validation Operator named `action_list_operator` would be part of the `great_expectations.yml` file.
+
+```yaml file=../../../tests/test_fixtures/configuration_for_testing_v2_v3_migration/spark/v2/great_expectations/great_expectations.yml#L57-L69
+```
+
+#### V3-Style Checkpoint
+
+Here is the equivalent configuration in V3-style. Notice that the Validation Operators have been migrated into the `action_list` field in the Checkpoint configuration. In addition, you will also need to remove the Validation Operations from `great_expectations.yml` as a manual step.  Also, notice the `batch_request` that refers to the data asset rather than `batch_kwargs`.  In our example, there are also some additional parameters like `ge_cloud_id` and `expectation_suite_ge_cloud_id` that are added automatically and do not need to be modified as part of the migration.
+
+For additional examples on how to configure V3-style checkpoints, including how to use `test_yaml_config` to build advanced configurations, please refer to our documentation here:
+
+- [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
+- [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
+
+
+```yaml file=../../../tests/test_fixtures/configuration_for_testing_v2_v3_migration/spark/v3/great_expectations/checkpoints/test_v3_checkpoint.yml#L1-L33
+```
+
+If the update was successful, then you should be able to see the updated Checkpoint `test_v3_checkpoint` by running `great_expectations checkpoint list`.
+
+```bash
+Using v3 (Batch Request) API
+Found 1 Checkpoint.
+ - test_v3_checkpoint
+ ```
+
+Finally, you can check if your migration has worked by running your new V3-style Checkpoint.
+
+```bash
+great_expectations checkpoint run test_v3_checkpoint
+```
+
+If everything is successful, then you should see output similar to below.:
+
+```bash
+Using v3 (Batch Request) API
+Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
+Calculating Metrics: 100%|█████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00,  6.67it/s]
+Validation succeeded!
+
+Suite Name                                   Status     Expectations met
+- Titanic.profiled                           ✔ Passed   2 of 2 (100.0 %)
+```
+
+</TabItem>
+</Tabs>
```

Comment:
This is a very long tab, and thanks to the styling in the docs, it's not clear when you're past the spark-specific section. I'll see about updating our css to frame tabs, I can see it all needs to be kept together

Reply:
oo thank you. BTW does it happen to be #4001 ? :)

Reply:
yup :)